src/PantryLink/PantryLink.react.js

```diff
@@ -0,0 +1,124 @@
+import classnames from 'classnames';
```

Comment:
Shouldn't these be named `Link.react.js` and `LinkBase.react.js` - there is no need to prefix them with `Pantry`. It will be up to the consumer to import it as something else if there is a naming conflict.